test/unit/org/apache/cassandra/cql3/CQLTester.java

```diff
@@ -471,16 +470,9 @@ public void beforeAndAfterFlush(CheckedFunction runnable) throws Throwable
 
     public void compact()
     {
-        try
-        {
-            ColumnFamilyStore store = getCurrentColumnFamilyStore();
-            if (store != null)
-                store.forceMajorCompaction();
-        }
-        catch (InterruptedException | ExecutionException e)
-        {
-            throw new RuntimeException(e);
-        }
+         ColumnFamilyStore store = getCurrentColumnFamilyStore();
+         if (store != null)
+             store.forceMajorCompaction();
```

Comment:
nit: I'm fine with cleaning this up, but it's unrelated to the main change, right?

Reply:
Just to confirm, should I return it there?
I am struggling a bit to find the borderline between - please clean stuff while you work somewhere and leave it as is so I am open for feedback :-)

Reply:
I'm fine with leaving the clean up in there.